freqtrade/persistence/trade_model.py

```diff
@@ -364,8 +364,6 @@ class LocalTrade:
     # percentage value of the initial stop loss
     initial_stop_loss_pct: Optional[float] = None
     is_stop_loss_trailing: bool = False
-    # stoploss order id which is on exchange
-    stoploss_order_id: Optional[str] = None
     # last update time of the stoploss order on exchange
     stoploss_last_update: Optional[datetime] = None
```

Comment:
@Axel-CH Now a very good question is - what do we do with `stoploss_last_update`?

in it's current state, it refers to `stoploss_order_id`. 
If we assume we can "X" stop orders, which one would it refer to? the oldest? newest?  don't think that's going to work (but removing this should be simple/easy, as it's really only used in 2 places (at least from an operational standpoint).

can an attribute in the order object replace this (most likely `order.order_date` ).

Reply:
I think using the most recent order_date of from all stoploss orders related to that trade is a good option